site-src/guides/implementers.md

```diff
@@ -98,6 +98,56 @@ For each HTTP request, the proxy CAN communicate the subset of endpoints the EPP
 
 The EPP communicates the chosen endpoint to the proxy via the `x-gateway-destination-endpoint` HTTP header and the `dynamic_metadata` field of the ext-proc response. Failure to communicate the endpoint using both methods results in a 503 error if no endpoints are ready, or a 429 error if the request should be dropped. The header and metadata values must match. In addition to the chosen endpoint, a single fallback endpoint CAN be set using the key `x-gateway-destination-endpoint-fallback` in the same metadata namespace as one used for `x-gateway-destination-endpoint`.
 
+### Implementing a Compatible Data Plane
+
+To conform with the Inference Extensions API, Gateway data planes must implement the [Endpoint Picker Protocol](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/004-endpoint-picker-protocol).
+
+At a high level, the protocol consists of metadata key/value pairs exchanged between the data plane and extensions containing relevant endpoint selection information:
+- From extension to data plane: the metadata contains the selected endpoints.
+- From data plane to extension: the metadata contains an optional subset of endpoints that the extension should pick from.
```

Comment:
Unfortunately mkdocs list formatting is pretty unforgiving. Generally need an empty line before any list starts: https://deploy-preview-1143--gateway-api-inference-extension.netlify.app/guides/implementers/#implementing-a-compatible-data-plane. Same comment applies throughout, will likely take some trial and error.
```suggestion

- From extension to data plane: the metadata contains the selected endpoints.
- From data plane to extension: the metadata contains an optional subset of endpoints that the extension should pick from.
```